lang/py3/avro/schema.py

```diff
@@ -37,10 +37,23 @@
  - A boolean;
  - Null.
 """
-
-
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+
+import sys
+if sys.version_info.major == 3:
```

Comment:
Maybe consider importing `six` here to make it a bit cleaner.

Reply:
I just didn't want to add yet another dependency

Reply:
Fair